ml.mdwalters.EightBall.appdata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>ml.mdwalters.EightBall</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>MIT</project_license>
+  <name>Eight Ball</name>
+  <summary>Make decisions easily</summary>
+	<description>
+	  <p>Eight Ball makes it more easier to make decisions, on whatever it is, from deciding if doing something is a good idea, or it's a bad idea!</p>
+	</description>
```

Comment:
as I previously said, this appstream file not sufficient...

missing `releases`, `screenshots`, `url`, `content_rating`

Reply:
still not done

Reply:
but it should actually be removed from here it's upstream.